Minecwaft.html

```diff
@@ -904,6 +904,18 @@
 		savebox.hidden = true;
 	}
 
+  function scode() {
+    //set a save code with a prompt, bit messy
+		var PLOAD = prompt('Enter Save Code:');
+    if (PLOAD === null) {
+        return;
+      }
+    localStorage.setItem('savecode', PLOAD);
```

Comment:
I'd rather not use localStorage for saves since there's a ~5,000,000 character quota limit shared between all of KA, and an hour or 2 of playing can easily exceed 100,000 characters. Evan Lewis said "If a project uses excessive amounts of storage... It may be hidden from the HL."

Reply:
at the time it is what I thought of, but I can see what you mean. I will join the discord when I'm able to, I appreciate the reply/info for the status